cognite/neat/rules/models/_rules/information_rules.py

```diff
@@ -465,29 +463,24 @@ def _as_dms_property(cls, prop: InformationProperty) -> "DMSProperty":
             nullable = None
         elif relation == "direct":
             nullable = True
-            container = ContainerEntity(prefix=prop.class_.prefix, suffix=prop.class_.suffix)
-            container_property = prop.property_
+            container, container_property = cls._get_container(prop)
         else:
-            container = ContainerEntity(prefix=prop.class_.prefix, suffix=prop.class_.suffix)
-            container_property = prop.property_
-
-        try:
-            return DMSProperty(
-                class_=prop.class_,
-                property_=prop.property_,
-                value_type=value_type,
-                nullable=nullable,
-                is_list=is_list,
-                relation=relation,
-                default=prop.default,
-                reference=prop.reference,
-                container=container,
-                container_property=container_property,
-                view=ViewPropEntity(prefix=prop.class_.prefix, suffix=prop.class_.suffix, version=prop.class_.version),
-                view_property=prop.property_,
-            )
-        except ValueError:
-            raise
+            container, container_property = cls._get_container(prop)
+
+        return DMSProperty(
+            class_=prop.class_,
+            property_=prop.property_,
+            value_type=value_type,
+            nullable=nullable,
+            is_list=is_list,
+            relation=relation,
+            default=prop.default,
+            reference=prop.reference,
+            container=container,
+            container_property=container_property,
+            view=ViewPropEntity(prefix=prop.class_.prefix, suffix=prop.class_.suffix, version=prop.class_.version),
```

Comment:
We assume here that Class id -> View id == Container id, which is something we need to highlight to end users, that in general modeling concepts there are no views or containers, but classes.